test/fixtures/cookbooks/acme_server/recipes/default.rb

```diff
@@ -87,16 +104,17 @@
   end
 end
 
-execute 'boulder_setup' do
+script 'boulder_setup' do
+  interpreter 'bash'
```

Comment:
needed to use script resource instead of execute in order to get "source" command